### PR TITLE
created student book

### PR DESCRIPTION
**Pull request by Rihovardini**

### CODE REVIEW

File: src/app/services/student-book-services/student-book.service.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class StudentBookService {
+  private URL = 'https://fierce-shore-32592.herokuapp.com/diaries';
```
Comment (dmitryy90) [inline src/app/services/student-book-services/student-book.service.ts]: wait for @tarasIV `s PR to fix this accordingly

File: src/app/student-book/student-book/student-book.component.scss [hunk 2]
```diff
@@ -0,0 +1,50 @@
+.weekSchedule {
+  margin-top: 70px;
+}
+.central {
+  text-align: center;
+  position: relative;
+  // padding-top: 1.3%;
```
Comment (dmitryy90) [inline src/app/student-book/student-book/student-book.component.scss]: commented code

Comment (Rihovardini) [reply src/app/student-book/student-book/student-book.component.scss]: fixed

File: src/app/services/student-book-services/student-book.service.ts (hunk 1 above)
Comment (Rihovardini) [reply src/app/services/student-book-services/student-book.service.ts]: @dmitryy90  Done!

Comment (dmitryy90) [reply src/app/services/student-book-services/student-book.service.ts]: must be '/diaries'

Comment (Rihovardini) [reply src/app/services/student-book-services/student-book.service.ts]: Yeah,I forgot to commit it

Comment (Rihovardini) [reply src/app/services/student-book-services/student-book.service.ts]: @dmitryy90  Done!

**Pull request merged**